src/Info.ts

```diff
@@ -46,7 +110,7 @@ export default class Info {
             response = infoData.message;
             response = response.replace(/{ddragonVersion}/g, this.versionChecker.ddragonVersion);
             response = response.replace(/{gameVersion}/g, this.versionChecker.gameVersion);
-            response = response.replace(/{counter}/g, infoData.counter.toString());
+            response = response.replace(/{counter}/g, infoData.counter ? infoData.counter.toString() : "0");
```

Comment:
Alternatively, `infoData.counter || "0"`, javascript will stringify automatically

Reply:
It doesn't like implicit stringify, Neither `infoData.counter || "0"` nor `infoData.counter || 0`.

It would have to be ``${infoData.counter || 0}`` or `(infoData.counter || 0).toString()`